src/client.rs

```diff
@@ -139,8 +144,15 @@ impl ClientFlow {
 
             match response {
                 Response::Status(status) => {
-                    self.maybe_abort_command(&status);
-                    break Some(ClientFlowEvent::StatusReceived { status });
+                    if let Some((tag, handle)) = self.maybe_abort_command(&status) {
+                        break Some(ClientFlowEvent::CommandRejected {
+                            handle,
+                            tag,
+                            status,
```

Comment:
Some arguments for keeping it:
- It probably doesn't matter, the user will ignore it or just log it
- If at some point in time we notice that there are serious servers that are seriously using `::No` for rejecting literals, then we can change `ClientFlow` so that it matches and returns `::No` without changing the API

I'd prefer to keep it and to revisit it later.

Reply:
Good point 👍🏻